torch/csrc/jit/python/init.cpp

```diff
@@ -719,7 +722,28 @@ void initJITBindings(PyObject* module) {
                 c10::nullopt));
           }
         }
-      });
+      })
+      .def("_jit_required_passes",
+          [](std::shared_ptr<Graph>& graph) {
+            Inline(*graph);
+            LowerSimpleTuples(graph);
+            ConstantPooling(graph);
+            runRequiredPasses(graph);
+            ConstantPropagation(graph);
+            EliminateDeadCode(graph);
+            EliminateCommonSubexpression(graph);
```

Comment:
How did you select these passes?
It would be great if you point out which code is relevant to this decision.

Reply:
I added a few more compilation passes and left some comments about where the code came from. Please check.